File: beginner/javascript/2060.js
var input = require("fs").readFileSync("stdin", "utf8");
var lines = input.split("\n");

var i, n, value, cont2 = 0, cont3 = 0, cont4 = 0, cont5 = 0;

n = parseInt(lines.shift())
value = lines.shift().split(" ");

for(i = 0; i < n; i++){

    if(value[i] % 2 == 0)
        cont2++;
    if(value[i] % 3 == 0)
        cont3++;
    if(value[i] % 4 == 0)
        cont4++;
    if(value[i] % 5 == 0)
        cont5++;
}

console.log(cont2 + " Multiplo(s) de 2");
console.log(cont3 + " Multiplo(s) de 3");
console.log(cont4 + " Multiplo(s) de 4");
console.log(cont5 + " Multiplo(s) de 5");

File: beginner/python/2060.py
cont2 = 0
cont3 = 0
cont4 = 0
cont5 = 0

n = int(input())

values = []

values = list(map(int, input().split()))

for i in range(0,n):

    if(values[i] % 2 == 0):
        cont2+=1
    if(values[i] % 3 == 0):
        cont3+=1
    if(values[i] % 4 == 0):
        cont4+=1
    if(values[i] % 5 == 0):
        cont5+=1


print("%d Multiplo(s) de 2" % cont2)
print("%d Multiplo(s) de 3" % cont3) 
print("%d Multiplo(s) de 4" % cont4)
print("%d Multiplo(s) de 5" % cont5)